<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>868</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>1274</width>
    <height>830</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Smartbody Debugger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QWidget" name="RenderView" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1264</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionConnect_Single"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSbm">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionData_Viewer"/>
    <addaction name="actionResource_Viewer"/>
    <addaction name="actionCommand_Window"/>
    <addaction name="actionFace_Viewer"/>
    <addaction name="actionBML_Creator"/>
    <addaction name="actionEnity_Creator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuSbm"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnectionStatus"/>
   <addaction name="actionToggleFreeLookCamera"/>
  </widget>
  <widget class="QDockWidget" name="sceneDockWidget">
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="sceneTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Scene</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QTreeWidget" name="sceneTree">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="rootIsDecorated">
            <bool>true</bool>
           </property>
           <property name="columnCount">
            <number>3</number>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="headerCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="headerDefaultSectionSize">
            <number>300</number>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string notr="true">2</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string notr="true">3</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="statisticsTab">
        <attribute name="title">
         <string>Statistics</string>
        </attribute>
        <widget class="QLabel" name="cameraPositionLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>211</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Camera Pos:</string>
         </property>
        </widget>
        <widget class="QLabel" name="networkFpsLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Network Fps:</string>
         </property>
        </widget>
        <widget class="QLabel" name="rendererFpsLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Renderer Fps:</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="utilsTab">
        <attribute name="title">
         <string>Utils</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="widgetResizable">
            <bool>false</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>479</width>
              <height>719</height>
             </rect>
            </property>
            <widget class="QPushButton" name="refreshButton">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>30</y>
               <width>61</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>Refresh</string>
             </property>
            </widget>
            <widget class="QLabel" name="selectedCharacterLabel">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>10</y>
               <width>141</width>
               <height>16</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Selected Character</string>
             </property>
            </widget>
            <widget class="QComboBox" name="selectedCharacterBox">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>30</y>
               <width>131</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QFrame" name="frame_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>60</y>
               <width>351</width>
               <height>221</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QLabel" name="animationNamesLabel">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>100</y>
                <width>91</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Animation Names</string>
              </property>
             </widget>
             <widget class="Line" name="line">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>30</y>
                <width>118</width>
                <height>3</height>
               </rect>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
             <widget class="QPushButton" name="QueryAnimButton">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>180</y>
                <width>75</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Query Anims</string>
              </property>
             </widget>
             <widget class="QComboBox" name="animationNamesBox">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>120</y>
                <width>291</width>
                <height>22</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="animationsLabel">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>81</width>
                <height>16</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Animations</string>
              </property>
             </widget>
             <widget class="QPlainTextEdit" name="animFilterBox">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>60</y>
                <width>291</width>
                <height>31</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="filterLabel">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>40</y>
                <width>71</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Search Filter</string>
              </property>
             </widget>
             <widget class="QPushButton" name="PlayAnimButton">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>150</y>
                <width>71</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Play</string>
              </property>
             </widget>
             <widget class="QPushButton" name="SetPostureButton">
              <property name="geometry">
               <rect>
                <x>90</x>
                <y>150</y>
                <width>71</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Set Posture</string>
              </property>
             </widget>
            </widget>
            <widget class="QFrame" name="frame_4">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>290</y>
               <width>351</width>
               <height>111</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QPushButton" name="GazeAtButton">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>80</y>
                <width>61</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Gaze At</string>
              </property>
             </widget>
             <widget class="QLabel" name="gazeTargetLabel">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>30</y>
                <width>61</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Gaze Target</string>
              </property>
             </widget>
             <widget class="QComboBox" name="gazeTargetBox">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>50</y>
                <width>131</width>
                <height>22</height>
               </rect>
              </property>
             </widget>
             <widget class="Line" name="line_5">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>118</width>
                <height>3</height>
               </rect>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
             <widget class="QLabel" name="controlLabel">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>0</y>
                <width>71</width>
                <height>16</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Control</string>
              </property>
             </widget>
             <widget class="QPushButton" name="StopGazeButton">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>80</y>
                <width>61</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Stop Gaze</string>
              </property>
             </widget>
            </widget>
            <widget class="QFrame" name="frame">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>410</y>
               <width>351</width>
               <height>221</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QComboBox" name="bmlFilesBox">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>160</y>
                <width>191</width>
                <height>22</height>
               </rect>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QLabel" name="ttsLabel">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>40</y>
                <width>91</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Text to Speech</string>
              </property>
             </widget>
             <widget class="QLabel" name="bmlFilesLabel">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>140</y>
                <width>51</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>BML Files</string>
              </property>
             </widget>
             <widget class="QPlainTextEdit" name="ttsBox">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>60</y>
                <width>191</width>
                <height>41</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="SpeakButton">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>110</y>
                <width>61</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Speak</string>
              </property>
             </widget>
             <widget class="Line" name="line_2">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>30</y>
                <width>118</width>
                <height>3</height>
               </rect>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
             <widget class="QLabel" name="speechLabel">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>71</width>
                <height>16</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Speech</string>
              </property>
             </widget>
             <widget class="QPushButton" name="runBmlButton">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>190</y>
                <width>61</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Run</string>
              </property>
             </widget>
            </widget>
            <widget class="QFrame" name="frame_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>640</y>
               <width>351</width>
               <height>91</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QLabel" name="renderingLabel">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>0</y>
                <width>71</width>
                <height>16</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Rendering</string>
              </property>
             </widget>
             <widget class="QCheckBox" name="showAxesBox">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>30</y>
                <width>81</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Show Axes</string>
              </property>
             </widget>
             <widget class="Line" name="line_3">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>118</width>
                <height>3</height>
               </rect>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
             <widget class="QCheckBox" name="showEyeBeamsBox">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>50</y>
                <width>111</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Show Eye Beams</string>
              </property>
             </widget>
             <widget class="QCheckBox" name="allowBoneUpdatesBox">
              <property name="geometry">
               <rect>
                <x>130</x>
                <y>30</y>
                <width>121</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>Allow Bone Updates</string>
              </property>
             </widget>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to a Smartbody Process</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionToggleFreeLookCamera">
   <property name="icon">
    <iconset>
     <normaloff>images/monkey_on_128x128.png</normaloff>images/monkey_on_128x128.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Free Look</string>
   </property>
   <property name="toolTip">
    <string>Toggle Free Look</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionResource_Viewer">
   <property name="text">
    <string>&amp;Resource Viewer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCommand_Window">
   <property name="text">
    <string>Command Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionData_Viewer">
   <property name="text">
    <string>D&amp;ata Viewer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionUtils_Window">
   <property name="text">
    <string>U&amp;tils Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionFace_Viewer">
   <property name="text">
    <string>Face Viewer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionBML_Creator">
   <property name="text">
    <string>&amp;BML Creator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionEnity_Creator">
   <property name="text">
    <string>Entity Creator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionConnect_Single">
   <property name="text">
    <string>Connect S&amp;ingle</string>
   </property>
   <property name="toolTip">
    <string>Connect to first sbm process found</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionConnectionStatus">
   <property name="icon">
    <iconset>
     <normaloff>../../images/ConnectionStatus_Disconnected.png</normaloff>../../images/ConnectionStatus_Disconnected.png</iconset>
   </property>
   <property name="text">
    <string>Connection Status</string>
   </property>
   <property name="toolTip">
    <string>Disconnected</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
